views/js/e2e/runner/_cleanup/cleanupCommands.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+import cleanupData from './cleanupData';
+import cleanupSelectors from './cleanupSelectors';
+
+/**
+ * Cleanup Commands
+ */
+Cypress.Commands.add('deleteImportedTest', () => {
```

Comment:
Function naming... is this only going to be used for an imported test or could it be a more generic command like `deleteTest(name_or_id_of_test)`?

Reply:
For now it is for one imported test which has to be used for all the testrunner related e2e tests. Later we can extend the functionality if needed.